share/smack/lib/math.c

```diff
@@ -6,195 +6,183 @@
 //Check the length of pointers
 //#if ( __WORDSIZE == 64 )
 #if defined(__LP64__) || defined(_LP64)
-#define BUILD_64   1
+#define BUILD_64 1
```

Comment:
Where is this used? Why do we need `BUILD_64`?

Reply:
This is currently used in smack.c to declare the lround functions [here](https://github.com/smackers/smack/blob/develop/share/smack/lib/smack.c#L1050). I'm not sure whether or not this is even needed, but it was there before I made any modifications.